### ISSUE TITLE
[Driftory website] Give people the ability to upload JSON files

### ISSUE THREAD

**Issue by Dan503** [pixfabrik/driftory-viewer#28]
It would be awesome if I could upload one of the exported JSON files and have it use that to generate the full comic. 

That would mean that if two artists are working on the same comic, they could share the workload by exporting the json file, send it to their friend, then their friend uploads the JSON to continue working on it. 

This would work great as long as they don't both work on the comic at the same time. Since comic editing is locked to their twitter accounts at the moment, unless they share a twitter account, multiple people are not able to work on the same comic sequence at the moment.

Note that for this to work well, you would need to be able to update an existing comic by uploading a json file and have it essentially replace the current Driftory data.

You would also need to be able to create a whole new comic based on a json export. The scenario here is that one artist has done the initial work and now the second artist needs to create a comic from the first artists json file.

**Comment by iangilman** [comment]
Good idea! I don't think this is top priority at the moment, but it's good to have in the to do list.
